en/docs/references/choreo-expression-syntax.md

```diff
@@ -0,0 +1,344 @@
+
+You can use the `+` operator to do string concatenations. The string value can come from a string literal or a variable.
+
+#### Examples
+
+ - Concatenating two string literals.
+  ```
+  "Hello " + "world!" 
+  ```
+ - Concatenating a string literal and a variable.
+  ```
+  "Hi " + nameVar
+  ```
+ - Concatenating two string variables.
+  ```
+  firstNameString + lastNameString
+  ```
+
+### Number Addition
+You can use the  `+` operator for number addition. The number values can come from a literal or a variable.
+
+#### Examples
+- Concatenating two integer literals.
+  ```
+  10 + 20
+  ```
+- Concatenating two decimal literals.
+  ```
+  0.56 + 1.84
+  ```
+- Concatenating a literal and a variable.
+  ```
+  100 + nameVariable
+  ```
+- Concatenating two variables.
+  ```
+  firstNumber + secondNumber
+  ```
+
+### Number Subtraction
+
+You can use the `-` operator for number subtraction. The number values can come from a literal or a variable.
+
+#### Examples
+- Subtracting two integer literals.
+  ```
+  10 - 20
+  ```
+- Subtracting two decimal literals.
+  ```
+  0.56 - 1.84
+  ```
+- Subtracting a literal and a variable.
+  ```
+  100 - <named variable>
+  ```
+- Subtracting two variables.
+  ```
+  <first number> - <second number>
+  ```
+
+### Number Division
+
+You can use the `/` operator for number division. The number values can come from a literal or a variable.
+
+#### Examples
+
+- Division between two integer literals.
+  ```
+  10 / 20
+  ```
+- Division between two decimal literals.
+  ```
+  0.56 / 1.84
+  ```
+- Division between a literal and a variable.
+  ```
+  100 / nameVariable
+  ```
+- Division between two variables.
+  ```
+  firstNumber - secondNumber
+  ```
+
+### Number Multiplication
+
+You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+
+#### Examples
+
+- Multiplying two integer literals.
+  ```
+  10 * 20
+  ```
+- Multiplying two decimal literals.
+  ```
+  0.56 * 1.84
+  ```
+- Multiplying a literal and a variable.
+  ```
+  100 * nameVariable
+  ```
+- Multiplying two variables.
+  ```
+  firstNumber * secondNumber
+  ```
+
+### Modulo Operation
+
+You can use the  `%` operator to get the remainder of a division. The number values can come from a literal or a variable.
+
+#### Examples
+- Modulus of two integer literals.
+  ```
+  10 % 4
+  ```
+- Modulus of a literal and a variable.
+  ```
+  100 % nameVariable
+  ```
+- Modulus of two variables.
+  ```
+  firstNumber % secondNumber
+  ```
+
+### Checking The Equality of Basic Types 
+
+In the [Ballerina language](https://ballerina.io/), basic types are `Strings`, `Integers`, `Decimal numbers`, and `Booleans`. You can compare the values of these types using the `==` binary operator. The values can come from a literal or a variable. Similarly `!=` operator is used to check the inequality. Note that the resulting value from these operators is always of Boolean type. 
```

Comment:
Since we are using code notations we should give types correctly. Then it should be `string`, `int`, `float`, `decimal`, and `boolean`.

Reply:
yeah, that makes sense.